releasenotes/notes/add-trace-agent-telemetry-endpoint-f398bcd47e637081.yaml

```
@@ -0,0 +1,12 @@
+# Each section from every release note are combined when the
+# CHANGELOG.rst is rendered. So the text needs to be worded so that
+# it does not depend on any information only available in another
+# section. This may mean repeating some details, but each section
+# must be readable independently of the other.
+#
+# Each section note must be formatted as reStructuredText.
+---
+features:
+  - |
+    APM: Add telemetry proxy endpoint to Trace Agent to allow Instrumentation Libraries to submit telemetry data
```

```suggestion
    APM: Add proxy endpoint to allow Instrumentation Libraries to submit telemetry data.
```
The `APM: ` prefix already makes it clear that it's the trace-agent.